WooCommerce/Classes/POS/ViewModels/PointOfSaleDashboardViewModel.swift

```diff
@@ -159,3 +159,10 @@ private extension PointOfSaleDashboardViewModel {
         case selfDeallocated
     }
 }
+
+// Conform to the protocol needed for unit testing
```

Comment:
We should be using protocols for real methods, not for testing private methods. We should be likely covering `PointOfSaleDashboardViewModel` dependencies with protocols, not the view model itself. More explanation here: https://github.com/woocommerce/woocommerce-ios/pull/13310#discussion_r1680493636